index.js

```diff
@@ -6,6 +6,52 @@ const testParameter = (name, filters) => {
 	return filters.some(filter => filter instanceof RegExp ? filter.test(name) : filter === name);
 };
 
+// From https://github.com/killmenot/valid-data-url/blob/8d56cab866469a4608001f0e8c5d73f65789ba13/index.js#L24
+// eslint-disable-next-line no-useless-escape
```

Comment:
If ESLint reports this, it's probably correct and you should remove the useless escape.